pkg/peer/peer.go

```diff
@@ -99,16 +99,17 @@ func (p *Peer[ID]) SubscribeTo(track common.TrackInfo) (*subscription.Subscripti
 // Writes the specified packets to the `trackID`.
 func (p *Peer[ID]) RequestKeyFrame(info common.TrackInfo) error {
 	// Find the right track.
-	track := p.state.GetRemoteTrack(info.TrackID, info.Layer)
+	track := p.state.GetRemoteTrack(info.TrackID, info.Simulcast.Layer)
 	if track == nil {
 		return ErrTrackNotFound
 	}
 
-	// The ssrc that we must use when sending the RTCP packet.
-	// Otherwise the peer won't understand where the packet comes from.
-	ssrc := uint32(track.SSRC())
+	if uint32(track.SSRC()) != info.Simulcast.SSRC {
+		return fmt.Errorf("track ssrc %d doesn't match the one in the info %d", track.SSRC(), info.Simulcast.SSRC)
+	}
 
-	rtcps := []rtcp.Packet{&rtcp.PictureLossIndication{MediaSSRC: ssrc}}
+	p.logger.Debugf("Keyframe request: %s (%s)", info.TrackID, info.Simulcast.Layer)
```

Comment:
Yeah, perhaps

Reply:
This is not a bad idea, though I think if we move it to trace, we probably would never enable it? I.e. ideally key frame requests should not be sent that often, I would say that they are always logged in big quantities, it would signify a bug. During the testing with debug logs I've noticed that they don't really happen that often (primarily during the switch/subscribe/unsubscribe), otherwise, if they were logged too often, there were always some bugs that caused it.

Does it feel like they would not be that bad for the debug logs since we only would use them during the **debugging**? In production, I think we would anyway run the SFU with the default maximum level of `info`. What do you think?

I think tracing is more suitable for things like... if we let's say logged each packet (there would be hundreds of them per second), whereas keyframe requests do not even happen every second.

Reply:
Done.